NEOSPlus/Logix/Color/Color to CMYK.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using FrooxEngine;
+using FrooxEngine.LogiX;
+using BaseX;
+
+[Category("LogiX/Color")]
+[NodeName("Color To CMYK")]
+public class ColorToCMYK : LogixNode
+{
+    public readonly Input<color> Color;
+    public readonly Output<float> C;
+    public readonly Output<float> M;
+    public readonly Output<float> Y;
+    public readonly Output<float> K;
+
+    protected override void OnEvaluate()
```

Comment:
What is the range for values in this node? Why is there a lot of arbitrary 255 and 100 multiplication and division?
Last I recall, the default range for colors in Neos is 0 to 1.

Reply:
should we hold off this node ? like this is just a conversion from rgba 255 each value to CYMK unless your thinking of the color number in the inspector that is 0-1